mage_ai/frontend/components/datasets/columns/ColumnReports.tsx

```diff
@@ -29,10 +29,15 @@ function ColumnReports({
 
   const featureSetStats = getFeatureSetStatistics(featureSet, featureUUID);
   const {
+    average,
     completeness,
     count,
     count_distinct: countDistinct,
     invalid_value_count: invalidValueCount,
+    max,
+    median,
+    min,
+    // mode,
```

Comment:
Is `mode` purposely left out for a reason?

Reply:
Yeah it didn't work so I thought we could do the metric later. My backends not returning it.